assembl/static2/js/app/components/home/objectives.jsx

```diff
@@ -41,7 +41,7 @@ class Objectives extends React.Component {
             </div>
             <div className="content-section">
               <div className="content-margin">
-                <Row className="center-content">
+                <Row className={classNames(haveImages ? null : 'center-content')}>
```

Comment:
You're using classNames wrong.
it should be `{classNames({'center-content': !haveImages}}`